cfme/utils/bz.py

```diff
@@ -172,6 +172,10 @@ def resolve_blocker(self, blocker, version=None, ignore_bugs=None, force_block_s
             version = current_version()
         if version == LATEST:
             version = bug.product.latest_version
+
+        if bug.resolution == 'DEFERRED':
```

Comment:
I think we need to actually do this check down on line ~215, when we are iterating over variants of the BZ.

It may be the case that the test is marked with BZ 123, which is for version x.y, but we're testing against version x.z, and cloned BZ 456 is the relevant BZ.  123's resolution doesn't matter in that case - 456 does.

Down where we are iterating over the filtered BZs and checking all their versions for fixes, we should be checking for deferred resolution as well.  I don't think it should go into `check_fixed_in` though.

Thoughts?

Reply:
Yeah. I admit I was trying to fix this in some fast but dirty way to make my numbers of passing tests nicer. I can spend some more time, but not too much as the tests affected by this are not in prio focus area.

Reply:
I think I have this solved.